code/modules/mob/living/carbon/xenomorph/castes/ravager/abilities_ravager.dm

```diff
@@ -122,6 +122,8 @@
 	atoms_to_ravage += get_step(owner, turn(owner.dir, -45)).contents
 	atoms_to_ravage += get_step(owner, turn(owner.dir, 45)).contents
 	for(var/atom/movable/ravaged AS in atoms_to_ravage)
+		if(ishitbox(ravaged) || isarmoredvehicle(ravaged)) //Handles APC/Tank stuff in a different for loop
+			break
```

Comment:
A) why is this breaking? B) why not just handle the ravaged here?  its also worth noting stuff like the mini tank dont have hitboxes

Reply:
Alright, I've updated to be done much better